common/authorization/claimMapper.go

```diff
@@ -29,13 +29,16 @@ package authorization
 import (
 	"crypto/x509/pkix"
 
+	"google.golang.org/grpc/credentials"
+
 	"go.temporal.io/server/common/service/config"
 )
 
 // Authentication information from subject's JWT token or/and mTLS certificate
 type AuthInfo struct {
-	authToken  string
-	tlsSubject *pkix.Name
+	AuthToken     string
+	TlsSubject    *pkix.Name
```

Comment:
[Nit] let's be consistent with below casing and call it TLSSubject instead?

Reply:
Fixed.